src/server/services/list/transformers/entity-to-entity-row.js

```diff
@@ -1,46 +1,40 @@
-import { config } from '~/src/config/config.js'
-import { formatText } from '~/src/config/nunjucks/filters/filters.js'
 import { statusTagClassMap } from '~/src/server/common/helpers/status-tag-class-map.js'
 import { noValue } from '~/src/server/common/constants/no-value.js'
 import {
   renderComponent,
   renderIcon
 } from '~/src/server/common/helpers/nunjucks/render-component.js'
+import startCase from 'lodash/startCase.js'
 
-function serviceToEntityRow(isAuthenticated) {
-  return (service) => {
-    const githubOrg = config.get('githubOrg')
-    const status = service?.serviceStatus?.status
-    const hasStatus = Boolean(status)
+function entityToEntityRow(isAuthenticated) {
+  return (entity) => {
+    const status = entity.status
 
-    const isFrontend = service.topics?.includes('frontend')
-    const isBackend = service.topics?.includes('backend')
-    const kind = isFrontend ? 'Frontend' : isBackend ? 'Backend' : false
-
-    const teams = service?.teams
+    const teams = entity?.teams
       ?.filter((team) => team.teamId)
       ?.map((team) => ({
         kind: 'link',
         value: team.name,
         url: `/teams/${team.teamId}`
       }))
 
-    // For services that are being created show, in-progress or failure tag. For created services show created date
-    const createdEntity = hasStatus
+    // For services that are being created show in-progress or failure tag. For created services show created date
+    const isCreatedSuccessfully = status === 'Success'
+    const createdEntity = isCreatedSuccessfully
       ? {
+          headers: 'created',
+          entity: { kind: 'date', value: entity.created }
+        }
+      : {
           headers: 'created',
           entity: {
             kind: 'tag',
-            value: formatText(status),
+            value: startCase(status),
```

Comment:
does this work ok with statuses with multiple words?

Reply:
yes. works for Success, InProgress & Failed